build/steps/test-core.yml

```diff
@@ -1,5 +1,14 @@
+parameters:
+  publishTestResults: false
+
 steps:
 - powershell: |
    dotnet test $(System.DefaultWorkingDirectory)\tests\Prism.Core.Tests\Prism.Core.Tests.csproj --configuration $env:BuildConfiguration --logger trx --no-build --collect "Code Coverage"
 
-  displayName: Test Prism.Core
\ No newline at end of file
+  displayName: Test Prism.Core
+
+- ${{ if eq(parameters.publishTestResults, true) }}:
```

Comment:
you shouldn't need this funky condition here... just use the parameter to set the enabled property of the task

Reply:
Can not believe I didn't think of that.